src/home.js

```diff
@@ -0,0 +1,32 @@
+export function loadInitialPage() {
```

Comment:
To much code repeated, try to break this function using some helper functions to avoid repetation.